contracts/src/v0.8/dev/ArbitrumValidator.sol

```diff
@@ -1,67 +1,70 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import "./interfaces/ArbitrumInboxInterface.sol";
 import "../interfaces/AggregatorValidatorInterface.sol";
-import "./interfaces/FlagsInterface.sol";
 import "../SimpleWriteAccessController.sol";
 
+/* dev dependencies - to be re/moved after audit */
```

Comment:
What do you mean to be removed? Won't the interfaces still be used by this contract post-audit?

Reply:
We use this `./dev` folder workflow for contracts in development. This means that we keep a copy of contracts under `./dev` and then move them outside `./dev` after the audit. This also means some contracts/deps will change their location after the move. So another commit is required after audit and the contract bytecode will potentially change, which is one of the reasons I'm not so fond of this type of workflow.

Here I just explicitly commented that after the audit these imports will change location because of the move from outside `./dev`.